In [15]:
import math

$\sigma=16$

$M_{(x)} = a$

$p=0.95$

$\overline{X}_{в}=80$

$n=256$

$$α = 1 - p = 1 - 0.95 = 0.05$$

Известно, что генеральная совокупность распределена нормально $\frac{\alpha}{2} = 0.025$

$[\overline{X}_{в} - z_{(\frac{\alpha}{2})} * \frac{\sigma}{\sqrt{n}}; \overline{X}_{в} + z_{(\frac{\alpha}{2})} * \frac{\sigma}{\sqrt{n}}]$

In [5]:
80-1.96*16/math.sqrt(256)

78.04

In [6]:
80+1.96*16/math.sqrt(256)

81.96

Этот интервал $[78.04; 81.96]$ с вероятностью 95% покрывает истенное $M_{(x)} = a$ генеральной совокупности 

$p=0.95$

$α = 1 - p = 1 - 0.95 = 0.05$

$n=10$

Известно, что генеральная совокупность распределена нормально $\alpha/2 = 0.025$

In [4]:
import statistics

In [5]:
x = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]
x_mean = statistics.mean(x) 
print (x_mean)

6.59


$\overline{X}_{в}=6.59$

In [6]:
import numpy as np

In [12]:
x = [6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1]
s2 = np.var(x, ddof=1)
print(s2)

0.2032222222222223


In [18]:
math.sqrt(s2)

0.4508017549014448

$p = 1 - \frac{\alpha}{2} = 1 - 0.025 = 0.975$

$K(\frac{\alpha}{2}) = t(\frac{\alpha}{2}) = 2.262$

In [20]:
6.59-2.262*0.45080/math.sqrt(10)

6.267539511206077

In [21]:
6.59+2.262*0.45080/math.sqrt(10)

6.912460488793923

Этот интервал $[6.2675; 6.9125]$ покрывает истинное значение X с доверительной вероятностью 95%.

$\mu_{0} = 17$

$\mu = 17.5$

$n = 100$

$D_{(x)} = 4$

$\alpha = 0.05$

$H_{0}:\mu = \mu_{0}$. Если $K_{(расч)} < K_{(таб)}$, принимаем гипотезу $H_{0}$ 

$H_{1}:\mu > \mu_{0}$. Если $K_{(расч)} > K_{(таб)}$, то отвергаем $H_{0}$ принимаем $H_{1}$

Известна $D_{(x)}$ из ГС можем найти $\sigma$ из ГС. $\sigma = \sqrt{D_{(x)}} = 2$

$K_{(расч)} = z_{(расч)} = \frac{\mu - \mu_{0}}{\frac{\sigma}{\sqrt{n}}}$

In [25]:
(17.5-17)/(2/math.sqrt(100))

2.5

$K_{(таб)} = z_{(таб)} = z(p=1-\alpha) = z(0.95) = 1.65 $

Вывод $z_{(расч)} > z_{(таб)}$ Отвергаем $H_{0}$ принимаем $H_{1}$

$\alpha = 0.01%$

$n =10$

$\mu_{0} = 200$

$\mu = 198.5$

$H_{0}:\mu = \mu_{0}$.

$H_{1}:\mu != \mu_{0}$.

In [26]:
x = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]
x_mean = statistics.mean(x) 
print (x_mean)

198.5


$K_{(расч)} = t_{(расч)} = \frac{|\mu - \mu_{0}|}{\frac{s}{\sqrt{n}}}$. В формуле учитывется модуль

Критерий $ K $ из табл.Стюдента. Стандартная ошибка среднего = $\frac{s}{\sqrt{n}}$, где $s$ среднеквадратическое отклонение от несмещенной оценки дисперсии выборки

In [30]:
x = [202, 203, 199, 197, 195, 201, 200, 204, 194, 190]
s2 = np.var(x, ddof=1)
print(s2)

19.833333333333332


In [31]:
math.sqrt(s2)

4.453463071962462

In [37]:
t = (abs(198.5-200))*4.45346/math.sqrt(10)
print(t)

2.1124615602680206


$t_{(расч)} = 2.1125$

$ t_{(таб)} = 2.821 $

Вывод $t_{(расч)} < t_{(таб)}$ принимаем $H_{0}$. Верно значение 200.